src/Nfts/ImageGallery/ImageGallery.tsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react'
+import {GestureResponderEvent, Image, ScrollView, StyleSheet, TouchableOpacity, View} from 'react-native'
+import {SafeAreaView} from 'react-native-safe-area-context'
+import SkeletonPlaceholder from 'react-native-skeleton-placeholder'
+import {useQuery} from 'react-query'
+
+import {Spacer} from '../../components'
+import {Text} from '../../components/Text'
+import {getAssetFingerprint} from '../../legacy/format'
+import {useSelectedWallet} from '../../SelectedWallet'
+import {YoroiNFT, YoroiNFTModerationStatus} from '../../yoroi-wallets/types'
+
+type Props = {
+  nfts: YoroiNFT[]
+  onNftPress: (index: number) => void
+}
+
+export const SkeletonGallery = (props: {amount: number} = {amount: 3}) => {
+  const array = [...new Array(props.amount)]
+  return (
+    <SafeAreaView edges={['left', 'right', 'bottom']} style={styles.safeAreaView}>
```

Comment:
do we need `SafeAreaView` in here?

Reply:
I'll move the `SafeAreaView` to the NFT gallery screen file